kadai1/task4233/eimg/eimg_test.go

```diff
@@ -0,0 +1,184 @@
+package eimg
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+)
+
+// TestEimg tests functions in eimg package as unittest.
+func TestEimg(t *testing.T) {
```

Comment:
テストを一つにまとめてるのって何か意図があります？
`go test`でTestHogeHogeは全部実施できるので、普通に3つのテストに分けたら良い気がしました。
```suggestion
func TestSetParameters...
func TestEncodeFile...
func TestConvertExtension...
```

Reply:
参考にしたライブラリがユニットテストをまとめていたので、それに倣っていました。
しかし、今回はまとめる必要がなさそうなので、ご指摘通り訂正しました。